jsonnet/thanos-mixin/records/querier.libsonnet

```diff
@@ -0,0 +1,91 @@
+{
+  prometheusRecords+:: {
+    groups+: [
+      {
+        name: 'thanos-querier.rules',
+        rules: [
+          {
+            record: 'thanos_querier:grpc_client_failures_per_unary:rate5m',
```

Comment:
```suggestion
            record: ':grpc_client_failures_per_unary:sum_rate',
```

Reply:
Should I keep the leading `:`? And I assume I have to change rest accordingly?

Reply:
NVM